src/matlab2tikz.m

```diff
@@ -4825,8 +4825,48 @@ function warnFacetedInterp(m2t)
             cbarStyleOptions = opts_add(cbarStyleOptions,...
                 'xticklabel pos', 'upper');
         case 'southoutside'
-
             cbarTemplate = 'horizontal';
+        case 'manual'
+            origUnits = handle.Units;
```

Comment:
Please use `get(handle,'Units')` syntax (and so on)
I know it looks uglier, but it is backwards compatible to Octave and MATLAB 2014a and older.